platformics/api/core/helpers.py

```diff
@@ -179,9 +230,10 @@ def get_aggregate_db_query(
     principal: Principal,
     where: dict[str, Any],
     aggregate: Any,
-    group_by: Optional[ColumnElement[Any]] = None,
```

Comment:
Was this input field just hanging around doing nothing before?

Reply:
This `group_by` used to be what is now the `remote` input to this function. We used it for dataloader aggregates to group the aggregate results by the top level id.